### DISCUSSION TITLE
Download strange percentage counter

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#215] [Q&A] [answered]
The percentage counter shows negative values over 100% while downloading and finally ends up decreasing to 100%
![grafik](https://github.com/devemux86/cruiser/assets/121371629/ad41e21a-8247-4b99-b126-18478cfd2666)

**Comment by devemux86** [comment]
@DolomitesTom thanks for the report!

Finally we see something, the change in percentage was a good idea.

The download progress comes from Android, I will check again what exactly we receive. :slightly_smiling_face:

**Comment by devemux86** [comment]
> shows negative values over 100%

@DolomitesTom do you mean the download was progressing correctly up to 100% on the Cruiser screen,
then the negative value appeared while Android notifications still had 100%?

**Comment by devemux86** [comment]
I think I found the reason, due to the size of the France map, some numerical conversion went wrong.

It will work properly in the next Cruiser update.

**Comment by devemux86** [comment, marked as answer]
Download progress for large files should work properly in [Cruiser 3.0.24](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
